charm/zookeeper/lib/charms/layer/zookeeper.py

```diff
@@ -44,7 +43,7 @@ def format_node(unit, node_ip):
 def get_package_version():
     with open('/snap/{}/current/meta/snap.yaml'.format(SNAP_NAME), 'r') as f:
```

Comment:
Should we use this instead?
```
        version = 'unknown'
        try:
            with open(
                '/snap/{}/current/meta/snap.yaml'.format(KAFKA_SNAP), 'r'
            ) as f:
                version = yaml.load(f).get('version')
        except (yaml.ReaderError, yaml.ParserError) as e:
            log(
                'unable to get version from kafka snap yaml: {}'.format(str(e))
            )
        finally:
            return version
```

Reply:
That's definitely more robust. Could also break this function out into a separate version.py file, as another cross-charm convention. I'll update.

Reply:
Actually, we can use `charms.layer.snap.get_installed_version` now.